<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BluetoothWindowClass</class>
 <widget class="QMainWindow" name="BluetoothWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bluetooth App</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="leftVertSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="BTGroupBox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QPushButton" name="pairingButton">
         <property name="text">
          <string>Pairing</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="powerButton">
         <property name="text">
          <string>Включен/Отключен</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="scanButton">
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="pairingLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="unPairingButton">
         <property name="text">
          <string>unPairing</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QListWidget" name="devicesListWidget"/>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="rightVertSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1115</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BluetoothWindow.qrc"/>
 </resources>
 <connections/>
</ui>
